_posts/en/newsletters/2024-09-27-newsletter.md

```diff
@@ -0,0 +1,267 @@
+  64-byte update.  This can allow thousands of token transfers to be
+  confirmed each time a new Bitcoin block is found by confirming only a
+  single Bitcoin transaction with an extra 64-byte data push.
+
+  The paper goes into details.  We found particularly interesting the
+  idea of trustlessly bridging bitcoin from the main blockchain to a
+  Shielded CSV (and back) without consensus changes using [BitVM][topic
+  acc], the use of accounts (section 2), the discussion about the effect
+  of blockchain reorganization on accounts and transfers (also section
+  2), the related discussion about depending on unconfirmed transactions
+  (section 5.2), and the list of possible extensions (appendix A).
+
+- **Draft of updated BIP process:** Mark "Murch" Erhardt
+  [posted][erhardt post] to the Delving Bitcoin mailing list to announce
+  the availability of a [pull request][erhardt pr] for a draft BIP that
+  describes an updated process for the BIP repository.  Anyone
+  interested in encouraged to review the draft and leave comments.  If
+  the community finds the final version of the draft acceptable, it will
+  become the process used by the BIP editors.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [What specific verifications are done on a fresh Bitcoin TX and in what order?]({{bse}}124221)
+  Murch enumerates the validity checks done by Bitcoin Core on a new transaction
+  when it is submitted to the mempool, including checks done in the
+  `CheckTransaction`, `PreChecks`, `AcceptSingleTransaction`, and related
+  functions.
+
+- [Why is my bitcoin directory larger than my pruning data limit setting?]({{bse}}124197)
+  Pieter Wuille notes that while the `prune` option limits the size of Bitcoin
+  Core's blockchain data, the chainstate, indexes, mempool backup, wallet files,
+  and other files are not subject to the `prune` limit and can grow in size
+  independently.
+
+- [What do I need to have set up to have `getblocktemplate` work?]({{bse}}124142)
+  User CoinZwischenzug also asks a [related question]({{bse}}124160) about how
+  to calculate the merkle root and coinbase transaction for a block. Answers to
+  both questions similarly indicate that while Bitcoin Core's `getblocktemplate`
+  can build candidate blocks of transactions and block header information, when
+  mining on non-test networks, coinbase transactions are created by mining or
+  [mining pool][topic pooled mining] software.
```

Comment:
Why is the asker credited here instead of the answerers? (and I'm not asking just because that includes me)

Reply:
Leaving this for @bitschmidty

Reply:
> Why is the asker credited here instead of the answerers? (and I'm not asking just because that includes me)

I thought the story was easier to tell. But youre probably right to note the answeroors as well.